repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -4514,5 +4514,27 @@ public void addHasLineage(Set<AtlasEdge> inputOutputEdges, boolean isRestoreEnti
         }
         RequestContext.get().endMetricRecord(metricRecorder);
     }
+    public void linkBusinessPolicy(String policyId, List<String> linkGuids) {
+        for (String guid : linkGuids) {
+            AtlasVertex ev = AtlasGraphUtilsV2.findByGuid(graph, guid);
+            if (ev != null) {
+                Set<String> existingValues = ev.getMultiValuedSetProperty("assetPolicyGUIDs", String.class);
```

Comment:
I guess this only required for writing assetPoliciesCount.
Instead, just fetch current assetPoliciesCount & add 1 & save it